docs/_quarto.yml

```diff
@@ -0,0 +1,106 @@
+project:
+  type: website
+  pre-render: version_config.py
```

Comment:
I think this pre-render script will run every time a page updates, when using `quarto preview` (which could slow down preview times a bit?)

https://quarto.org/docs/projects/scripts.html#pre-and-post-render

Reply:
It looks like that is the case! I think I'm going to leave it in the pre-render section for now to ensure it is always run (primarily to build the docs locally) since then you can use `quarto preview`, `quarto render`, or `make docs` and the version will be updated regardless. But this is still great to note if the docs site gets huge (or maybe when it gets merged with vetiver.rstudio.com), this is a way to speed up `quarto preview` if it feels sluggish.